src/test/java/io/thorntail/example/GreetingServiceTest.java

```diff
@@ -13,56 +13,50 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
+package io.thorntail.example;
 
-package io.openshift.boosters;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.swarm.arquillian.DefaultDeployment;
 
 import javax.ws.rs.client.Client;
 import javax.ws.rs.client.ClientBuilder;
 import javax.ws.rs.client.WebTarget;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 
-import org.jboss.arquillian.container.test.api.RunAsClient;
-import org.jboss.arquillian.junit.Arquillian;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.wildfly.swarm.Swarm;
-import org.wildfly.swarm.arquillian.CreateSwarm;
-import org.wildfly.swarm.arquillian.DefaultDeployment;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 
-/**
- * @author Heiko Braun
- */
 @RunWith(Arquillian.class)
 @DefaultDeployment
 public class GreetingServiceTest {
-
     @Test
     @RunAsClient
-    public void testResource() {
+    public void resource() {
```

Comment:
Not sure why the `test` prefix is removed, is it a new convention (`Test` is already in part of the class name) ?

Reply:
Or it is probably to do with the method order

Reply:
The `test*` convention comes from pre-annotations JUnit. These days, I consider it poor style.

Reply:
I don't this test relies on method order. If it did, it would either be using Arquillian's `@InSequence`, or the methods would be called `_01_...` / `_02_...` etc.